src/components/html/EcApplication.html

```diff
@@ -156,6 +202,18 @@ <h3>{{ title }}</h3>
 
     <div class="col-sm">
       <div class="card">
+        <div class="alert alert-warning" v-if="application.newVersionAvailable">
+          <div class="row d-flex align-items-center">
+            <div class="col-md-1 justify-content-center">
+              <i class="fa fa-warning" style="font-size: 32px;"> </i>
+            </div>
+            <div class="col-md-10">
+              {{ i19newVersionAvailableMsg}}
+              <br />
+              <strong>{{ i19attention }}:</strong> {{ i19warningToReinstallAppMsg }}!
```

Comment:
```suggestion
              <strong>{{ i19attention }}:</strong> {{ i19warningToReinstallAppMsg }}
```

É melhor que o `...Msg` já contenha as pontuações porque isso também vai variar entre as linguagens posteriormente.